Task6_2.py

```diff
@@ -0,0 +1,21 @@
+import random
```

Comment:
Jeśli ten skrpyt wykonuje jakieś konkretne zadanie , w tym przypadku (Skrypt wykonujący 'GuessNumber') nazwa pliku powinna to odwzierciedlać